pyccl/halos/halo_model.py

```diff
@@ -384,4 +440,5 @@ def I_0_22(self, cosmo, k, a, prof, *,
             uk34 = prof34_2pt.fourier_2pt(
                 cosmo, k, self._mass, a, prof3, prof2=prof4).T
 
+        # TODO: Is this k ordering correct?
```

Comment:
is it?

Reply:
It depends on what's the default for CCL. When building my 2D arrays, I am doing the following:
```
k = k_use[:, None]
kp = k_use[None, :]
```
so the second index correspond to k'. This is not what is happening here. However, this works if the profiles of `uk12` and `uk34` are the same. What should I change? This line or the others lines?

Reply:
It's really up to you. If you prefer it this way, just remove the comment (or change it to specify what's going on)

Reply:
Done. Left as it was for Tkk_1h to avoid changing the default ordering (i.e. (k', k)).